activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -293,6 +293,9 @@ def delete(key)
       super(convert_key(key))
     end
 
+    def except(*keys)
+      slice(*self.keys - keys.map { |key| convert_key(key) })
```

Comment:
Same as previous comment:

It should be safe to call `map!` here, due to the splat on `keys`.